### DISCUSSION TITLE
fetch error

### DISCUSSION THREAD

**Discussion by fandyajpo** [vercel/next.js#54388] [App Router]
```
const getProductst = async () => {
  const res = await fetch(
    "https://64e495b5c555638029137451.mockapi.io/product"
  );
  return res.json();
};

const HomePage = async () => {
  const products = await getProductst();

  console.log(products);

  return <div>HomePage</div>;
};

export default HomePage;
```

i got this error when i fetch my mockAPI, but if i fetch other api, it looks fine, how to solve this

mockAPI : https://64e495b5c555638029137451.mockapi.io/product
other API : https://dummyjson.com/users?limit=1&skip=1
<img width="1440" alt="Screenshot 2023-08-22 at 23 32 59" src="https://github.com/vercel/next.js/assets/91927970/1e8b5a66-988f-433f-8287-d4d51e8e321c">

**Comment by katiesun138** [comment]
From what I see, your MockAPI is not in a JSON format. It seems to be a json within a List, whereas the other API renders the data in JSON format. 
Another suggestion is to await res.json() instead of returning it right away since the method returns a promise, so you will need to await it.

**Reply by fandyajpo** [reply]
bro bro ive got the solution, i just upgrade my nodejs to the latest and it work perfectly

20.2.0 -> 20.5.1

thanks for your response :)

if i add try catch and throwing err in getProduct function, it throw me this
```
{
  "cause": {
    "code": "ERR_SOCKET_CONNECTION_TIMEOUT"
  }
}
```
https://stackoverflow.com/questions/76179568/socket-connection-timout-error-in-node-js

so i decide to upgrade my nodejs
